devito/operator/profiling.py

```diff
@@ -1,18 +1,20 @@
+import cgen as c
```

Comment:
to be moved before the `cached_property` import. This one is not from the python lib